SleepEngine/Engine/GameTemplate/global_shortcuts.h

```diff
@@ -19,6 +19,21 @@ inline ResourceManager& globalResourceManager()
     return Game::instance().getResourceManager();
 }
 
+inline Camera& globalCamera()
+{
+    return Game::instance().getCamera();
+}

```

Comment:
are you sure we really need these functions?
In my opinion, it is better to write 'Game::instance().getCamera'

Reply:
I still think they are cool. It's shorter, clearer and users don't need to know how the camera is stored, just that it is global static